### PR TITLE
Exercise 13

### PR DESCRIPTION
**Pull request by Tayoto-Mikaella**
Please review my file. Thank you

### CODE REVIEW

File: tayoto_mikaellaantonette/index.html [hunk 1]
```diff
@@ -86,28 +86,28 @@ <h2>My Goals</h2>
...
+                <div>
+                  <button type="submit" 
+                  class="submit-goal" 
+                  id="comment_button">
+                    Submit
+                  </button>
+                </div>
+                <div>
+                  <button id="sort-asc">Sort by Date (ASC)</button>
+                  <button id="sort-desc">Sort by Date (DESC)</button>
```
Comment (Villa-VillaDiana) [inline tayoto_mikaellaantonette/index.html:107]: ID should be written in snake case.

Reference:
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

File: tayoto_mikaellaantonette/index.html [hunk 2]
```diff
@@ -86,28 +86,28 @@ <h2>My Goals</h2>
...
+                </div>
+                <ul id="comment-list"></ul>
```
Comment (Villa-VillaDiana) [inline tayoto_mikaellaantonette/index.html:109]: ID should be written in snake case.

Reference:
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

File: tayoto_mikaellaantonette/script.js [hunk 3]
```diff
@@ -1,17 +1,63 @@
+const form = document.querySelector("form");
+const nameInput = document.querySelector("name");
+const commentInput = document.querySelector("gcomment");
+const commentList = document.querySelector("comment-list");
+const sortAscButton = document.querySelector("sort-asc");
+const sortDescButton = document.querySelector("sort-desc");
```
Comment (Villa-VillaDiana) [inline tayoto_mikaellaantonette/script.js:6]: Your comment section is not working; it seems like you are using the wrong syntax here. If you are referring to an ID, it should be a hash character followed by your ID name.

`const nameInput = document.querySelector("#name");`

Reference: https://www.w3schools.com/html/html_id.asp#:~:text=Using%20The%20id%20Attribute&text=It%20is%20also%20used%20by,properties%20within%20curly%20braces%20%7B%7D.

File: tayoto_mikaellaantonette/script.js [hunk 4]
```diff
@@ -1,17 +1,63 @@
...
 }
 
 function name() {
-  let name = document.getElementById("name");
-  if (name.value.length > 0) {
-    document.getElementById("comment_button").disabled = false;
-  } else {
-    document.getElementById("comment_button").disabled = true;
-  }
+  comment();
```
Comment (Villa-VillaDiana) [inline tayoto_mikaellaantonette/script.js:41]: Are you using this function? If not please remove it.

**Review by Villa-VillaDiana — CHANGES_REQUESTED** [review]

**Review by Villa-VillaDiana — CHANGES_REQUESTED** [review]
I  forgot to mention this earlier: you can have the '/>' on the same line as your link. Please don't rely too much on Prettier.

``

**Review by Villa-VillaDiana — APPROVED** [review]
Nice! You can now merge this :)

**Comment by Tayoto-Mikaella** [comment]
Thank you

**Pull request merged**